get() works on composite GIDCollections

stinebuu opened the pull request:
Expanded get on SLI level to handle Composite GIDCollections.

stinebuu committed:
get() on composite GIDCollections

hakonsbm commented on lib/sli/nest-init.sli:
```diff
@@ -125,17 +166,15 @@ def
 
     GetStatus dup size 1 neq
-    { % if not a layer
+    { % if not a layer 
```
Remove trailing whitespace.

hakonsbm commented on pynest/nest/lib/hl_api_types.py:
```diff
@@ -281,10 +281,17 @@ def get(self, *params, **kwargs):
                 if pandas_output:
                     try:
+                        # This try will probably soon be removed, once we
+                        # deside what to do with get and GC's with metadata.
```
deside -> decide

hakonsbm commented on pynest/nest/tests/test_GIDCollection.py:
```diff
@@ -593,6 +593,54 @@ def test_get_pandas(self):
+
+        # Check that calling get with string works on composite gc's, both on
+        # parameters all the models have, and on individual parameters.
+        Cm_ref = [x*1. for x in range(251, 259)]
```
Add whitespace around `*`.

hakonsbm reviewed (changes requested):
Looks pretty good, there's only some formatting and a typo that has to be fixed.

stinebuu committed:
Formatting

stinebuu committed:
Merge remote-tracking branch 'compNeuroNMBU_nest/nest-3' into
nest3-getOnComposite

stinebuu committed:
Merge remote-tracking branch 'compNeuroNMBU_nest/nest-3' into
nest3-getOnComposite

stinebuu committed:
Workaround with layers until get only concerns nodes. get('local') does
not work on layers at the moment, so tests failed,once we have syntax
for GIDCollection with metadata, this will be solved.

stinebuu committed:
pep8
